src/vendor/Path.ts

```diff
@@ -3,6 +3,7 @@ import * as sys from "deno/path/mod.ts"
 import * as fs from "deno/fs/mod.ts"
 import { PlainObject } from "is_what"
 import {readLines} from "deno/io/buffer.ts"
+import slash from "https://deno.land/x/slash@v0.3.0/mod.ts";
```

Comment:
For version control purposes, add `"slash": "https://deno.land/x/slash@v0.3.0/mod.ts",` to `import-map.json` and use `import slash from "slash"`.

Reply:
Thanks for the comment.  I committed this change.